### ISSUE TITLE
Deploying WAR services is not working

### ISSUE THREAD

**Issue by leonardofl** [choreos/choreos_middleware#16]
At the end of the /tmp/chef-client.log:

```
FATAL: Chef::Exceptions::Exec: package[tomcat6] (tomcat::default line 37) had an error: Chef::Exceptions::Exec: apt-get -q -y install tomcat6=6.0.32-5ubuntu1.3 returned 100, expected 0
```

**Comment by leonardofl** [comment]
Solved by pbmoura: 7a5b641cb153865adb725cedb72fb4e6c77a9676
Ubuntu version changed to the last LTS version.

**Comment by leonardofl** [comment]
Today the java recipe did not work in the apt-get install line.
Probably Nelson was right: apt-get update is necessary!
